images/collector.Dockerfile

```diff
@@ -37,7 +37,7 @@ RUN ln -s /usr/bin/python3 /usr/bin/python
 # Install pip
 RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py
 
-ARG GOLANG_VERSION=1.19.4
+ARG GOLANG_VERSION=1.19.9
 ARG GOLANG_SHA256=c9c08f783325c4cf840a94333159cc937f05f75d36a8b307951d5bd959cf2ab8
```

Comment:
I think you also need to  update the SHAs

Reply:
Yeah just realized...

Reply:
Fixed.